tests/cephfs/cephfs_mirroring/test_cephfs_mirroring_modify_remote_snapshots.py

```diff
@@ -0,0 +1,609 @@
+            cephfs_mirror_node[0], source_clients[0], source_fs, snap_count
+        )
+        if validate_syncronisation:
+            log.error("Snapshot Synchronisation failed..")
+            raise CommandFailed("Snapshot Synchronisation failed")
+
+        log.info(
+            "Fetch the daemon_name, fsid, asok_file, filesystem_id and peer_id to validate the synchronisation"
+        )
+        fsid = fs_mirroring_utils.get_fsid(cephfs_mirror_node[0])
+        log.info(f"fsid on ceph cluster : {fsid}")
+        daemon_name = fs_mirroring_utils.get_daemon_name(source_clients[0])
+        log.info(f"Name of the cephfs-mirror daemon : {daemon_name}")
+        asok_file = fs_mirroring_utils.get_asok_file(
+            cephfs_mirror_node[0], fsid, daemon_name
+        )
+        log.info(f"Admin Socket file of cephfs-mirror daemon : {asok_file}")
+        filesystem_id = fs_mirroring_utils.get_filesystem_id_by_name(
+            source_clients[0], source_fs
+        )
+        log.info(f"filesystem id of {source_fs} is : {filesystem_id}")
+        peer_uuid = fs_mirroring_utils.get_peer_uuid_by_name(
+            source_clients[0], source_fs
+        )
+        log.info(f"peer uuid of {source_fs} is : {peer_uuid}")
+
+        log.info("Validate if the Snapshots are syned to Target Cluster")
+        result_snap_k1 = fs_mirroring_utils.validate_snapshot_sync_status(
+            cephfs_mirror_node[0],
+            source_fs,
+            snap1,
+            fsid,
+            asok_file,
+            filesystem_id,
+            peer_uuid,
+        )
+        result_snap_f1 = fs_mirroring_utils.validate_snapshot_sync_status(
+            cephfs_mirror_node[0],
+            source_fs,
+            snap2,
+            fsid,
+            asok_file,
+            filesystem_id,
+            peer_uuid,
+        )
+        if result_snap_k1 and result_snap_f1:
+            log.info(f"Snapshot '{result_snap_k1['snapshot_name']}' has been synced:")
+            log.info(
+                f"Sync Duration: {result_snap_k1['sync_duration']} of '{result_snap_k1['snapshot_name']}'"
+            )
+            log.info(
+                f"Sync Time Stamp: {result_snap_k1['sync_time_stamp']} of '{result_snap_k1['snapshot_name']}'"
+            )
+            log.info(
+                f"Snaps Synced: {result_snap_k1['snaps_synced']} of '{result_snap_k1['snapshot_name']}'"
+            )
+
+            log.info(f"Snapshot '{result_snap_f1['snapshot_name']}' has been synced:")
+            log.info(
+                f"Sync Duration: {result_snap_f1['sync_duration']} of '{result_snap_f1['snapshot_name']}'"
+            )
+            log.info(
+                f"Sync Time Stamp: {result_snap_f1['sync_time_stamp']} of '{result_snap_f1['snapshot_name']}'"
+            )
+            log.info(
+                f"Snaps Synced: {result_snap_f1['snaps_synced']} of '{result_snap_f1['snapshot_name']}'"
```

Comment:
Most of the steps above are common across all CephFS Mirror Automations, can we plan for cephfs_mirror_setup module in mirrorring lib that can perform this instead, so that it can be reusable for next set of automations?

Reply:
Yeah, I have discussed the same with Amar too. Will do in next PR.